src/node/internal/internal_buffer.ts

```diff
@@ -2294,6 +2294,21 @@ export function isUtf8(value: ArrayBufferView) {
   return bufferUtil.isUtf8(value);
 }
 
+export function transcode(source: ArrayBufferView, fromEncoding: string, toEncoding: string) {
+  if (!isArrayBufferView(source)) {
+    throw new ERR_INVALID_ARG_TYPE('source', 'ArrayBufferView', typeof source);
+  }
+  const normalizedFromEncoding = normalizeEncoding(fromEncoding);
+  if (!Buffer.isEncoding(normalizedFromEncoding)) {
+    throw new ERR_UNKNOWN_ENCODING(fromEncoding);
+  }
+  const normalizedToEncoding = normalizeEncoding(toEncoding);
+  if (!Buffer.isEncoding(normalizedToEncoding)) {
+    throw new ERR_UNKNOWN_ENCODING(toEncoding);
+  }
+  return bufferUtil.transcode(source, normalizedFromEncoding, normalizedToEncoding);
```

Comment:
The return value of `bufferUtil.transcode(...)` here is going to be an `ArrayBuffer`. This needs to be wrapped in a `Buffer`. So you'll want something like `return Buffer.from(bufferUtil.transcode(...` instead